ui/revenuecatui/src/main/kotlin/com/revenuecat/purchases/ui/revenuecatui/components/stack/StackComponentView.kt

```diff
@@ -80,58 +91,119 @@ internal fun StackComponentView(
     )
 
     if (stackState.visible) {
-        val backgroundColorStyle = stackState.backgroundColor?.let { rememberColorStyle(scheme = it) }
-        val borderStyle = stackState.border?.let { rememberBorderStyle(border = it) }
-        val shadowStyle = stackState.shadow?.let { rememberShadowStyle(shadow = it) }
+        if (style.badge != null) {
+            when (style.badge.style) {
```

Comment:
Just a heads-up since this changed recently: `style.badge.style` should probably be part of `stackState` so it can participate in the overrides mechanism.